utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java

```diff
@@ -74,23 +76,59 @@ private static void createPullSecret() {
         }
     }
 
+    /**
+     * Deploys the operator and the custom resource.
+     */
     private static void deployUsingOperator() {
         log.info("Deploying using Operator");
-        if (!TestUtils.isUserAdmin()) {
-            StringBuilder sb = new StringBuilder("\n");
-            sb.append("****************************************************\n");
-            sb.append("* Operator deployment needs user with admin rights *\n");
-            sb.append("****************************************************\n");
-            sb.append(
-                "If you are using minishift, you can use \"oc adm policy --as system:admin add-cluster-role-to-user cluster-admin developer\"\n");
-            log.error(sb.toString());
-            throw new RuntimeException(sb.toString());
-        }
 
-        deployCrd();
         deployOperator();
         deploySyndesisViaOperator();
-        TodoUtils.createDefaultRouteForTodo("todo2", "/");
+    }
+
+    /**
+     * Grants the permissions via the admin user to the regular user.
+     */
+    public static void grantPermissions() {
+        String operatorImage = TestConfiguration.syndesisOperatorImage();
+
+        log.info("Pulling operator image {}", operatorImage);
+        ProcessBuilder dockerPullPb = new ProcessBuilder("docker",
+            "pull",
+            operatorImage
+        );
+
+        try {
+            dockerPullPb.start().waitFor();
+        } catch (Exception e) {
+            log.error("Could not pull operator image", e);
+            fail("Failed to pull operator");
+        }
```

Comment:
So we moved this from `getOperatorResources` here, that seems to me like replacing one non-systematic solution with another. Maybe refactor into a separate method and run before any calls to the operator?

Reply:
or maybe call it just once in SyndesisTemplate.deploy() before any other method related to the operator?

Reply:
Yep, works for me as well